airflow/providers/google/cloud/triggers/bigquery.py

```diff
@@ -113,6 +116,14 @@ async def run(self) -> AsyncIterator[TriggerEvent]:  # type: ignore[override]
                         self.poll_interval,
                     )
                     await asyncio.sleep(self.poll_interval)
+        except asyncio.CancelledError:
```

Comment:
Would it be posible for us to add an unit test for this case?

Reply:
Added the test.